### PR TITLE
Desktop support

### PR DESCRIPTION
**Pull request by MasterLeiva**
- Desktop layout
- Flexbox
- Grid
- Use of media queries
- Accessibility

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -162,17 +167,40 @@ <h2>About Myself</h2>
...
+            type="text" 
+            class="input-container-2" 
+            placeholder="Write your message here..." 
+            maxlength="500" 
+            name="comment" 
+            id="comment" 
+            required
+          ></textarea>
           <label class="labels" for="comment">Comment</label>
           <button type="submit" class="button-form">Get in touch</button>
```
Comment (stanleySimeon) [inline index.html]: Greeting!
You did an amazing job, but there is a little thing you'll need to work on to go through the next steps.

According to the figma design, I suggest you center this button to make your design match the Desktop version in the Figma design.

> The Figma Dsign
>
>Screenshot 2023-06-08 at 5 02 23 PM
>
> ## Your Design
>
>Screenshot 2023-06-08 at 5 02 56 PM

File: index.html [hunk 2]
```diff
@@ -61,20 +61,21 @@ <h3>CANOPY</h3>
...
             <p>2015</p>
           </div>
-          <p class="cards-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <p class="cards-p">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
           <ul class="tags">
             <li><a>HTML</a></li>
+            <li><a>Ruby on rails</a></li>
             <li><a>CSS</a></li>
             <li><a>JavaScript</a></li>
           </ul>
```
Comment (stanleySimeon) [inline index.html:81]: Please, make sure you stick to the design as much as possible `(e.g., font, colors, images, text, margins)` using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Kindly, I suggest you keep the mobile design exactly as it is in the Figma template when you update the Desktop version. 😄

> ## The Figma Design
>
>Screenshot 2023-06-08 at 5 07 30 PM
>
> ## Your Design
>
>Screenshot 2023-06-08 at 5 07 56 PM
>Screenshot 2023-06-08 at 5 08 58 PM
>Screenshot 2023-06-08 at 5 10 27 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @MasterLeiva, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MasterLeiva 

Your project is complete! Great job 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design on point
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**